src/Compilers/CSharp/Test/Emit/CodeGen/CodeGenRefConditionalOperatorTests.cs

```diff
@@ -0,0 +1,799 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+using Microsoft.CodeAnalysis.CSharp.Test.Utilities;
+using Roslyn.Test.Utilities;
+using Xunit;
+
+namespace Microsoft.CodeAnalysis.CSharp.UnitTests.CodeGen
+{
+    public class CodeGenRefConditionalOperatorTests : CSharpTestBase
```

Comment:
We can do that later. While the feature in active development it feels simpler to have tests grouped together.

---
In reply to: [105259055](https://github.com/dotnet/roslyn/pull/17436#discussion_r105259055) [](ancestors = 105259055)